### ISSUE TITLE
emoji search not working 

### ISSUE THREAD

**Issue by Krzmbrzl** [vector-im/element-android#2606]
**Describe the bug**
The emoji search triggered by typing a colon does not work.

**To Reproduce**
1. Type a colon
2. Type e.g. "cur"

**Expected behavior**
The popped up emoji list should (only) contain the thinking face emoji (referenced as :curious: in the web app).

Note that this is independent of the actual emoji chosen.

**Smartphone :**
 - Device: Shiftphone Shift6m
 - OS:  Android 8.0

**Additional context**
 - App version and store: 1.0.11 [40100110] (F-dffdcfe1) F-droid
 - Homeserver: matrix.org

**Comment by bmarty** [comment]
Emoji search is working for me, and return more results than Element Web because we also search in keywords related to the emoji, which is a nice feature IMHO.

`:cur` does not find the same Emoji than Element Web, and to me the issue is more an issue to be reported on Element Web, because "curious" is actually the Emoji for "Thinking face". See https://emojipedia.org/thinking-face/. You will find it in Element Android by typing `:think`

**Comment by Krzmbrzl** [comment]
> You will find it in Element Android by typing :think

No this still doesn't work.

I start to type `:` at which point the emoji list appears. The list does not change at all though if I start typing characters. So this issue is not about the `:curious:` emoji in particular.

I noticed however that if I e.g. type `:think`, the insert a space (at which point the list disappears) and delete it again, at the time my cursor reaches the end of `:think` again (after having deleted the space again), the list re-appears but this time it is filtered according to what I have typed.

**Comment by bmarty** [comment]
OK, strange. On my phone (running Android 10), the list of completion is updated each time I type (or delete) a char after `:`. Do you have the same pb with completion for room members? You can try it by typing `@` in a room with lots of members.

**Comment by Krzmbrzl** [comment]
> Do you have the same pb with completion for room members?

No. For mentions the list updates as expected while typing the name :shrug:
